### ISSUE TITLE
`ibm_is_vpc`: Allow to specify region

### ISSUE THREAD

**Issue by ganhuang** [IBM-Cloud/terraform-provider-ibm#867]
### Terraform Version
```
Terraform v0.11.11
+ provider.ibm v0.19.0
```

### Affected Resource(s)
- ibm_is_vpc

From https://ibm-cloud.github.io/tf-ibm-docs/v0.19.0/r/is_vpc.html, currently, the provider doesn't support to specify `region` for vpc creations (generation 1). Therefore all the vpcs will be created in us-south region. 

But we're able to specify the region from IBM portal.

**Comment by ganhuang** [comment]
When the region is added to provider configuration:

```
provider "ibm" {
  version         = "0.19.0"
  generation      = "1"
  region          = "jp-tok"
}
```

It errors out with:

```
Error: Error applying plan:

4 error(s) occurred:

* module.vpc-jp-tok.ibm_is_network_acl.public_acl: 1 error(s) occurred:

* ibm_is_network_acl.public_acl: Error occured while fetching auth key for vpc: "ServiceEndpointDoesnotExist: UAA endpoint doesn't exist for region: \"jp-tok\""
* module.vpc-jp-tok.ibm_is_vpc.vpc: 1 error(s) occurred:

* ibm_is_vpc.vpc: Error occured while fetching auth key for vpc: "ServiceEndpointDoesnotExist: UAA endpoint doesn't exist for region: \"jp-tok\""
* module.vpc-jp-tok.ibm_is_network_acl.default_acl: 1 error(s) occurred:

* ibm_is_network_acl.default_acl: Error occured while fetching auth key for vpc: "ServiceEndpointDoesnotExist: UAA endpoint doesn't exist for region: \"jp-tok\""
* module.vpc-jp-tok.ibm_is_network_acl.private_acl: 1 error(s) occurred:

* ibm_is_network_acl.private_acl: Error occured while fetching auth key for vpc: "ServiceEndpointDoesnotExist: UAA endpoint doesn't exist for region: \"jp-tok\""

Terraform does not automatically rollback in the face of errors.
Instead, your Terraform state file has been partially updated with
any resources that successfully completed. Please address the error
above and apply again to incrementally change your infrastructure.
```

**Comment by hkantare** [comment]
We are looking into this

**Comment by ruusnak** [comment]
Any idea when this will be fixed? We need to provision VPCs in frankfurt gen1 infra, and all my VPCs go to Dallas. The documentation is still missing the option to specify the region

**Comment by hkantare** [comment]
its already fixed in v0.19.1 release...

**Comment by ruusnak** [comment]
Thanks, just realized it obeys the provider setting
____________________________________________

Ystävällisin terveisin / With kind regards
Jouko Ruuskanen
Senior IT Architect
IBM Cloud


P.O. Box 265, 00101 Helsinki, Finland
Käyntiosoite: Tietokuja 2, 00330 Helsinki
Tel. 4591 GSM +358-40-7328140
E-mail: jouko.ruuskanen@fi.ibm.com
